docs/commands/lint.md

```diff
@@ -24,7 +24,7 @@ redocly lint --version
 
 | Option                 | Type     | Description                                                                                                                                                                          |
 | ---------------------- | -------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
-| apis                   | [string] | Array of API description filenames that need to be linted. See [the Apis section](#apis) for more options.                                                                           |
+| apis                   | [string] | Array of API or Arazzo description filenames that need to be linted. See [the Apis section](#apis) for more options.                                                                 |
```

Comment:
@lornajane maybe we might need more renames here in the docs as we can pass Arazzo description file here.

Reply:
More mentions updated, thx.